web/src/test/java/it/pagopa/selfcare/dashboard/web/controller/ProductControllerTest.java

```diff
@@ -108,4 +114,48 @@ void retrieveProductBackoffice() throws Exception {
         verifyNoInteractions(productServiceMock);
     }
 
+    @Test
+    void getProductBrokers() throws Exception {
+        // given
+        BrokerInfo brokerInfo = new BrokerInfo();
+        brokerInfo.setBrokerCode("code");
+        String productId = "prod-pagopa";
+        String institutionType = "PSP";
+        when(brokerServiceMock.findAllByInstitutionType(anyString()))
+                .thenReturn(List.of(
+                        brokerInfo
+                ));
+        // when
+        MvcResult result = mvc.perform(MockMvcRequestBuilders
+                        .get(BASE_URL + "/{productId}/brokers/{institutionType}", productId, institutionType)
+                        .contentType(APPLICATION_JSON_VALUE)
+                        .accept(APPLICATION_JSON_VALUE))
+                .andExpect(status().isOk())
+                .andReturn();
+        // then
+        Collection<BrokerResource> resources = objectMapper.readValue(result.getResponse().getContentAsString(),
+                new TypeReference<>() {
+                });
+        assertNotNull(resources);
```

Comment:
you could test also if actual broker attribute are equals to expected brokerInfo.